taxonomies/categories.txt

```diff
@@ -96315,7 +96315,7 @@ fr:Saucisses végétariennes de Vienne, saucisses végétales de Vienne
 nl:Vegetarische weense worstjes
 
 <en:Plant-based foods
-<en:Dairy substitutes
+<en:Dairy substitutes, Dairies substitutes
```

Comment:
the synonym must be on the entry itself, not on the line that indicates that there is a parent